src/libraries/System.Numerics.Tensors/src/System/Numerics/Tensors/netcore/ReadOnlyTensorSpan.cs

```diff
@@ -522,10 +522,11 @@ public void CopyTo(scoped TensorSpan<T> destination)
             {
                 scoped Span<nint> curIndexes;
                 nint[]? curIndexesArray;
-                if (Rank > 6)
+                if (Rank > 5)
```

Comment:
nit: would be good to make this a named constant so we don’t need to update a million places if we refactor the size

Reply:
Good idea. Let me go ahead and do that.

Reply:
Fixed.